packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/editable-tile-item/EditableTileItem.spec.tsx

```diff
@@ -11,7 +11,12 @@ describe('EditableTileItem component unit test suite', () => {
 
     // when
     const { getByTestId, getByText } = render(
-      <EditableTileItem label={label} onClickEdit={() => {}} />,
+      <EditableTileItem
+        urn="urn"
+        iamActions={[]}
+        label={label}
+        onClickEdit={() => {}}
```

Comment:
It's required to give empty function ? We cannot just no define`onClickEdit` ?

Reply:
`onClickEdit` is not an optional props on this component

Reply:
`onClickEdit` is not an optional props on this component